.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "editor.rulers": [100]
```

Comment:
I added this .vscode config as a guide to avoid long comment lines that broke lint